SecondAssignmentFirstProgram.cs

```diff
@@ -0,0 +1,178 @@
+
+
+/*
+
+Program Created By Ragu Balagi Karuppaiah.
+Date : 21. 06. 2019
+
+Second Assignment - FirstProgram (Tavisca)
+
+
+*/
+
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FirstAssignment

```

Comment:
Create a class in a separate file. You can create a static class with static methods, so that you do not need to create an instance of the class.